### PR TITLE
Improve system docs

### PR DESCRIPTION
**Pull request by xPi2**
## Dework Task

https://app.dework.xyz/nation3/court?taskId=22508aa5-bc14-443c-9144-ac50efcd6b99

### CODE REVIEW

**Review by gregfromstl — APPROVED** [review]

**Review by pythonpete32 — APPROVED** [review]

File: docs/frameworks/collateral-agreement.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+The standard finalization procedure requires all the parties to agree to finalize the contract. If the agreement is disputed, the arbitrator will be the only one capable of finalizing the agreement.
+
+To finalize a disputed agreement, the arbitrator can submit a settlement that adjusts the participants' positions in the agreement.
+
+After the agreement is finalized, all participants can withdraw from it, recovering the collateral in the balance of their position at finalization.
+
+**Practical example:**
+
+- Bob and Alice join a collateral agreement with three tokens as collateral each.
```
Comment (aahna-ashina) [inline docs/frameworks/collateral-agreement.md:24]:
```suggestion
- Bob and Alice join a collateral agreement with three tokens each as collateral.
```

**Review by aahna-ashina — APPROVED** [review]

File: docs/agreement-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Each [agreement framework](./architecture#agreement-framework) will define specific finalization requirements. Once those requirements are met, the agreement's enforceability will end, and the parties will be free of their contractual obligations.
+
+For example, a collateral agreement could define that only when all the parties consent to finalize the agreement can they recover their deposited collateral.
+
+## Dispute creation & resolution
+
+Any of the parties involved can dispute the agreement in Court. When disputed, the parties involved will need to provide allegations & evidence, in most cases making the terms of the agreement accessible to the jurors resolving the dispute.
+
+*(Further documentation on how to submit allegations & evidences to the Court to be defined)*
```
[the remark this answers is not in the document]
Comment (xPi2) [reply docs/agreement-lifecycle.md:46]: I wanted to have it as a visible comment

**Review by aahna-ashina — APPROVED** [review]

**Review by pythonpete32 — APPROVED** [review]

File: docs/frameworks/collateral-agreement.md [hunk 3]
```diff
@@ -23,6 +23,6 @@ After the agreement is finalized, all participants can withdraw from it, recover
 
 - Bob and Alice join a collateral agreement with three tokens each as collateral.
 - They have agreed to include in the terms that; if any of the parties breach the agreement, depending on the obligations that are not met, they will compensate the other party with between half and the totality of their collateral.
-- Bob doesn't fulfill his contractual obligations, and Alice disputes the agreement.
+- Bob doesn't fulfil his contractual obligations, and Alice disputes the agreement.
```
Comment (aahna-ashina) [inline docs/frameworks/collateral-agreement.md:26]: @pythonpete32 For consistency, would it be better if we stick with US English throughout all of our Nation3 documentation/communications? `¯\_(ツ)_/¯`

**Review by aahna-ashina — APPROVED** [review]

File: docs/frameworks/collateral-agreement.md (hunk 3 above)
Comment (pythonpete32) [reply docs/frameworks/collateral-agreement.md:26]: American spelling always trips me up. I didn't even realise this was a thing

**Review by pythonpete32 — APPROVED** [review]

**Pull request closed without merging**